src/oc_erchef/apps/data_collector/rebar.lock

```diff
@@ -1,20 +1,23 @@
-[{<<"goldrush">>,
-  {git,"git://github.com/DeadZen/goldrush.git",
-       {ref,"71e63212f12c25827e0c1b4198d37d5d018a7fec"}},
-  1},
- {<<"ibrowse">>,
-  {git,"git://github.com/opscode/ibrowse.git",
-       {ref,"8f3f6a3a30730b193cc340a8885a960586dc98de"}},
-  1},
+[
+ %% these apps are not present in the rebar.conf so
+ %% commenting out for now to see what happens.
+%{<<"goldrush">>,
+% {git,"git://github.com/DeadZen/goldrush.git",
+%      {ref,"57e71f2789632a085f0963e1244644fdcb1941b9"}},
+% 1},
+%{<<"ibrowse">>,
+% {git,"git://github.com/chef/ibrowse.git",
```

Comment:
If we find out eventually we're not using this then we should move that repo over to the chef-boneyard org.

Reply:
Sure, or just delete entirely.

Reply:
Usually we just move them just in case